### ISSUE TITLE
Noticed new pop up windows in Pokemon Go

### ISSUE THREAD

**Issue by ca2014mp2** [finndev/PokeBuddy#781]
I noticed that in the newest version .33 that they have added more pop up windows to incite a response that the bot will not cover. Perhaps that is what is triggering the stops from disappearing or getting soft booted... just a thought.

**Comment by Rawr2134** [comment]
Good idea, i didn't really think about that...could be the speed warning thing, needs to be under 20km/h?

**Comment by curtiola** [comment]
I typically keep it below 12, someone recommended it in an issue yesterday sometime too, no issues for me so far
